docs/tracing.md

```diff
@@ -0,0 +1,143 @@
+# Tracing
+
+Tracing across the stack follows, as much as possible, the [Open Telemetry]
+specifications. Configuration environment variables are specified in the
+[OpenTelemetry Environment Variable Specification].
+
+We use the [opentelemtry-go] package, which currently does not have default support
+for the `OTEL_TRACES_EXPORTER` environment variables. Therefore, we provide some
+helper functions under [`boxo/tracing`](../tracing/) to support these.
+
+In this document, we document the quirks of our custom support for the `OTEL_TRACES_EXPORTER`,
+as well as examples on how to use tracing, create traceable headers, and how
+to use the Jaeger UI. The [Gateway examples](../examples/gateway/) fully support Tracing.
+
+- [Environment Variables](#environment-variables)
+  - [`OTEL_TRACES_EXPORTER`](#otel_traces_exporter)
+  - [`OTLP Exporter`](#otlp-exporter)
+  - [`Jaeger Exporter`](#jaeger-exporter)
+  - [`Zipkin Exporter`](#zipkin-exporter)
+  - [`File Exporter`](#file-exporter)
+  - [`OTEL_PROPAGATORS`](#otel_propagators)
+- [Using Jaeger UI](#using-jaeger-ui)
+- [Generate `traceparent` Header](#generate-traceparent-header)
+
+## Environment Variables
+
+For advanced configurations, such as ratio-based sampling, please see also the
+[OpenTelemetry Environment Variable Specification].
```

Comment:
missing URL?

Reply:
Not missing, they're on the bottom. It works with Markdown.

Reply:
slick, i didn't know MD did this